### ISSUE TITLE
Improvement: assign nameserver based on those used on the host

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#1493]
Related to #1316 

An improvement to the experience of using `--network-ipaddress` is to prevent having to specify the nameserver. In most cases it will be OK to use the same nameserver as those used on the host.

You could use
```
PS> Get-DnsClientServerAddress | Select-Object –ExpandProperty ServerAddresses
```

but as you can see, the complexity is in using the right one as given from the command.  On my system I have the following as a result:

```
10.208.0.245
10.200.0.246
10.208.0.246
10.72.17.5
10.68.5.26
fec0:0:0:ffff::1
fec0:0:0:ffff::2
fec0:0:0:ffff::3
10.0.21.1
208.67.220.220
8.8.8.8
fec0:0:0:ffff::1
fec0:0:0:ffff::2
fec0:0:0:ffff::3
```

IPv6 can be excluded, but all the others are all valid. However, we can only pass 2 nameservers to the VM.

  * [x] ~~add nameservers after simple reachability (ping?)~~
  * [x] ~~add nameservers after DNS lookup (to determine if external lookup is possible and to add nameservers that denied to respond to the ICMP ping)~~
  * [x] ~~add message on startup to "Determining nameserver to use. Consider setting network-nameserver to skip this check"~~

**Comment by gbraad** [comment]
Although the message and script already support multiple nameservers, it might be suggested to also have a look at the related task: #1497

**Comment by gbraad** [comment]
```
C:\> ipconfig /all | findstr /R "DNS\ Servers"
```

this only works on English distros

**Comment by gbraad** [comment]
https://github.com/gorillalabs/go-powershell/issues/3

```
Get-DnsClientServerAddress | ^
    Where-Object { $_.InterfaceAlias -like 'vEthernet*' } | ^
    ForEach-Object { $_.ServerAddresses }
```

**Comment by LalatenduMohanty** [comment]
Resolved via #1849
